### PR TITLE
Add div_precision_increment part document

### PR DESCRIPTION
**Pull request by yibin87**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v8.0 (TiDB 8.0 versions)
- [ ] v7.6 (TiDB 7.6 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: system-variables.md [hunk 1]
```diff
@@ -461,6 +461,16 @@ mysql> SELECT * FROM t1;
...
 
+### `div_precision_increment` <span class="version-mark">从 v8.0.0 版本开始引入</span>
+
+- 作用域：SESSION | GLOBAL
+- 是否持久化到集群：是
+- 是否受 Hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value) 控制：是
+- 类型：整数型
+- 默认值：`4`
+- 范围：`[0, 30]`
+- 这个变量用于控制要增加使用运算符执行的除法运算结果的小数位数。
```
Comment (hfxsd) [inline system-variables.md:472]:
```suggestion
- 这个变量用于控制使用运算符执行的除法运算结果的小数位数。
```

File: system-variables.md [hunk 2]
```diff
@@ -461,6 +461,16 @@ mysql> SELECT * FROM t1;
...
 
+### `div_precision_increment` <span class="version-mark">从 v8.0.0 版本开始引入</span>
+
+- 作用域：SESSION | GLOBAL
+- 是否持久化到集群：是
+- 是否受 Hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value) 控制：是
+- 类型：整数型
+- 默认值：`4`
+- 范围：`[0, 30]`
+- 这个变量用于控制使用运算符执行的除法运算结果的小数位数。
```
Comment (hfxsd) [inline system-variables.md:472]:
```suggestion
- 这个变量用于控制使用运算符 `/` 执行的除法运算结果的小数位数。
```

File: system-variables.md [hunk 3]
```diff
@@ -461,6 +461,16 @@ mysql> SELECT * FROM t1;
...
 
+### `div_precision_increment` <span class="version-mark">从 v8.0.0 版本开始引入</span>
+
+- 作用域：SESSION | GLOBAL
+- 是否持久化到集群：是
+- 是否受 Hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value) 控制：是
+- 类型：整数型
+- 默认值：`4`
+- 范围：`[0, 30]`
+- 这个变量用于控制使用运算符 `/` 执行的除法运算结果的小数位数。
```
Comment (windtalker) [inline system-variables.md:472]: 这里应该是“除法运算结果增加的小数位数”吧？如果原来是 decimal(10,2) 的话，除法结果的小数位数会变成 2 + div_precision_increment 吧？

Comment (yibin87) [reply system-variables.md:472]: 嗯 这边描述的有点笼统，我改下吧

**Review by windtalker — APPROVED** [review]
LGTM

File: system-variables.md [hunk 4]
```diff
@@ -461,6 +461,16 @@ mysql> SELECT * FROM t1;
...
 
+### `div_precision_increment` <span class="version-mark">从 v8.0.0 版本开始引入</span>
+
+- 作用域：SESSION | GLOBAL
+- 是否持久化到集群：是
+- 是否受 Hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value) 控制：是
+- 类型：整数型
+- 默认值：`4`
+- 范围：`[0, 30]`
+- 这个变量用于控制使用运算符 `/` 执行的除法运算结果增加的小数位数。
```
Comment (joey-yez) [inline system-variables.md:472]:
```suggestion
- 这个变量用于控制使用运算符 `/` 执行的除法运算结果的小数位数，该功能与 MySQL 保持一致。
```

File: system-variables.md [hunk 5]
```diff
@@ -461,6 +461,16 @@ mysql> SELECT * FROM t1;
...
 
+### `div_precision_increment` <span class="version-mark">从 v8.0.0 版本开始引入</span>
+
+- 作用域：SESSION | GLOBAL
+- 是否持久化到集群：是
+- 是否受 Hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value) 控制：是
+- 类型：整数型
+- 默认值：`4`
+- 范围：`[0, 30]`
+- 这个变量用于控制使用运算符 `/` 执行的除法运算结果的小数位数，该功能与 MySQL 保持一致。
```
Comment (Oreoxmt) [inline system-variables.md:472]:
```suggestion
- 这个变量用于控制使用运算符 `/` 执行除法操作时，结果保留的小数位数。该功能与 MySQL 保持一致。
```

**Review by Oreoxmt — APPROVED** [review]

**Review by Oreoxmt — APPROVED** [review]

**Review by hfxsd — APPROVED** [review]

**Review by joey-yez — APPROVED** [review]

**Pull request merged**